### PR TITLE
NSP-330: Added mobile view of planned downtime

### PR DESCRIPTION
**Pull request by davidmthornton**

### CODE REVIEW

File: app/views/helpers/PlannedDowntimeTableMobile.scala.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <th scope="col" class="govuk-table__header">@messages("planned-downtime.start")</th>
+        <td class="govuk-table__cell govuk-!-text-align-right">
+            @formatDate(downtime.startDate)<br />
+            @formatTime(downtime.startTime)
+        </td>
+    </tr>
+    <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">@messages("planned-downtime.end")</th>
+        <td class="govuk-table__cell govuk-!-text-align-right">
+            @formatDate(downtime.endDate)<br />
```
Comment (gorkemunsal) [inline app/views/helpers/PlannedDowntimeTableMobile.scala.html:49]: not sure if we should use <br/> on the views

File: conf/planned-downtime.conf [hunk 2]
```diff
@@ -17,4 +17,9 @@
 # affectedChannel is one of gbArrivals, xiArrivals, gbDepartures or xiDepartures
 #     { "startDate": "YYYY-MM-DD", "startTime": "HH:MM", "endDate": "YYYY-MM-DD", "endTime": "HH:MM", "affectedChannel": "gbArrivals" }
 
-planned-downtime = []
+planned-downtime = [
```
[the remark this answers is not in the document]
Comment (davidmthornton) [reply conf/planned-downtime.conf]: Thanks I've removed this - it should be empty

File: app/views/helpers/PlannedDowntimeTableMobile.scala.html (hunk 1 above)
Comment (davidmthornton) [reply app/views/helpers/PlannedDowntimeTableMobile.scala.html:49]: `<br>`

Comment (davidmthornton) [reply app/views/helpers/PlannedDowntimeTableMobile.scala.html:49]: didn't know github actually lets you html in the comments!

**Pull request merged**